target/linux/ramips/image/mt7621.mk

```
@@ -368,56 +368,44 @@ define Device/iodata_wn-ax1167gr
 endef
 TARGET_DEVICES += iodata_wn-ax1167gr
 
-define Device/iodata_wn-ax1167gr2
+define Device/iodata_nand
+  DEVICE_VENDOR := I-O DATA
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   UBINIZE_OPTS := -E 5
-  UIMAGE_MAGIC := 0x434f4d42
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 51200k
-  DEVICE_VENDOR := I-O DATA
-  DEVICE_MODEL := WN-AX1167GR2
   LOADER_TYPE := bin
   KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | lzma | uImage lzma
```

Please save a few CPU cycles on the second `lzma` encoding!

```suggestion
  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | lzma -a0 | uImage lzma
```